OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -451,7 +451,12 @@ public override bool Tick(Actor self)
 
 				if (progress >= Distance)
 				{
-					mobile.SetCenterPosition(self, To);
+					var toPos = To;
+					// apply ramp offset to ground units
```

Comment:
Once our CI runs, it'll raise a style rule issue here - "Single-line comment should be preceded by blank line".

Anything that shows as a warning will be prompted to an error when run in CI, so you'll need to fix those.

Your IDE should show the warnings - but you can also run `make check` locally.